deepl-connector/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.axonivy.connector.deepl</groupId>
+  <artifactId>deepl-connector</artifactId>
+  <version>10.0.0-SNAPSHOT</version>
+  <packaging>iar</packaging>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>com.axonivy.ivy.ci</groupId>
+        <artifactId>project-build-plugin</artifactId>
+        <version>10.0.1</version>
+        <configuration>
+            <ivyVersion>10.0.4</ivyVersion>
```

Comment:
we should have a property <useLatestMinor>true</useLatestMinor>

I think we would use that for all market artifacts. And we also would need this for our own build infrastructure, that we no longer need to define maven ranges.
Maybe a good task for tgr?

Reply:
yes, good idea, if we can easily get that information from the website, sure.
I create a story for TGR

Reply:
https://axonivy.atlassian.net/browse/XIVY-10786

Reply:
The infos are already available.